cmd/did-rest/startcmd/start.go

```diff
@@ -188,6 +198,24 @@ func startDidService(parameters *didRestParameters, srv server) error {
 	return srv.ListenAndServe(parameters.hostURL, constructCORSHandler(router))
 }
 
+func createVDRI() (vdriapi.Registry, error) {
+	var opts []vdripkg.Option
+
+	keyVDRI, err := key.New()
+	if err != nil {
+		return nil, fmt.Errorf("failed to create key vdri: %w", err)
+	}
+
+	opts = append(opts, vdripkg.WithVDRI(keyVDRI))
```

Comment:
@aholovko any reason why it isn't already in the aries framework's defaults?

Reply:
@llorllale I think that we should do it. If everyone agrees I will create a ticket for that.

Reply:
@aholovko I agree

Reply:
@llorllale I created ticket [#1847](https://github.com/hyperledger/aries-framework-go/issues/1847) for adding key vdri to framework defaults, as it looks to be a useful functionality. But here, I will probably leave the existing logic as we are not using the "full" aries framework, only the vdri component.

Thank you @fqutishat again for clarifying that part to me.